api/controllers/OpenEAppController.js

```diff
@@ -96,7 +100,31 @@ const OpenEAppController = {
     },
 
     _formatDate(date) {
-        return moment(date).format('DD MMMM YYYY');
+        return dayjs(date).format('DD MMMM YYYY');
+    },
+
+    _calculateDaysLeftToDownload(req) {
+        const urlObj = new URL(
+            `${req.protocol}://${req.headers.host}${req.originalUrl}`
+        );
+        const queryParams = new URLSearchParams(urlObj.search);
```

Comment:
what stops a user from changing the queryParams themselves so they're able to get to their documents? or is that not a risk?

Reply:
Good point. Nothing stops the user from changing the query params, the query param will only be used for the notification, changing it won't give users access to their documents if the download has expired.

But I've just had a thought which means I might be able to do it without a query param. I'm going to try it out :)